config/root/clusterworkspacetype-organization.yaml

```diff
@@ -5,3 +5,9 @@ metadata:
 spec:
   initializers:
   - tenancy.kcp.dev/organization
+  defaultChildWorkspaceType: Universal
+  allowedChildWorkspaceTypes:
+  - Team
+  - Universal
+  allowedParentWorkspaceTypes:
+  - Organization
```

Comment:
This must be some implicit, reserved type Root

Reply:
I don't understand what this means.

Reply:
This is the restriction for the Org workspace type. Currently, the yaml say "Orgs can only live in Orgs", i.e. there is no base-case (which would be Root).

Reply:
That's a strange behaviour that root is special. Why do we do this?

Also, we don't want Org in Org.